astropy/modeling/tests/test_core.py

```diff
@@ -231,20 +232,28 @@ def test_render_model_2d():
                 if (im is None) & (xy is None):
                     # this case is tested in Fittable2DModelTester
                     continue
-                actual = render_model(model, arr=image, coords=xy)
+                actual = render_model(model, arr=im, coords=xy)
+                if im is None:
+                    assert_allclose(actual, render_model(model, coords=xy))
                 # assert images match
-                assert_allclose(expected, actual, atol=2e-7)
-                # assert flux conserved
-                assert ((np.sum(expected) - np.sum(actual)) / np.sum(expected)) < 1e-7
+                assert_allclose(expected, actual, atol=5e-7)
+                # assert model fully captured
+                if (x0, y0) == (70, 35):
+                    boxed = render_model(model)
+                    flux = np.sum(expected)
+                    assert ((flux - np.sum(boxed)) / flux) < 1e-7
+    try:
```

Comment:
Ok, I see. You can test for exceptions like this:
https://github.com/astropy/astropy/blob/master/astropy/convolution/tests/test_kernel_class.py#L472